src/components/body/DocTool/TestDocumentForm/TestDocumentForm.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from "react";
+import PageTitle from "../../../modules/PageTitle/PageTitle";
+import HorizontalCard from "../../../modules/HorizontalCard/HorizontalCard";
```

Comment:
Is this import needed here?

Reply:
Actually not, my bad, fixing it, good point  !